### ISSUE TITLE
[Screen Reader - Forms Pro-Invitation Details] On the Invitation page, after pressing spacebar on the sorting key of the table, Screen Reader is not providing the sorting information.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#8235]
Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
![mas1 3 1-screen reader is not providing the check and unchecked information](https://user-images.githubusercontent.com/30456910/53938579-b2e45c00-40d6-11e9-9cdf-6e65b6e73954.png)

User Experience:

If, the Screen Reader is not announcing the sorting information of the table after pressing spacebar on the column header of the table, then the user might get affected. Hence, failed to get the sorting information of the table.

Pre-Requisite:

Survey questions should be added using above steps.

### Environment Information 

OS Build: Version 1809 (OS build 17763.253)
Browser: Edge, Chrome
Browser: Edge, Chrome

### Please provide a reproduction of the bug in a codepen:
 Repro Steps:

1. Open URL https://service.projectshimla.azurewebsites.net/Pages/DesignPage.aspx and sign in using valid credentials.
2. Forms homepage appears  and skip the See how it works screens 

3. Navigate to 'Create new Survey' menu present at the left side of the homepage and select it.​

4. 'Create New Survey' dropdown is opened. then Select 'New Survey' . ​

5. Navigate to View Responses tab and select it.​

6. Responses tab with overview tab selected opens, navigate to Invitations tile and select it.

7. Navigate with the Narrator and reach the column header (Date) of the table.

8. Press spacebar on the column header (Date) to perform sorting functionality.

9. Verify, whether Narrator announce the sorting information or not.

#### Actual behavior:
 
On the Invitation page, after pressing spacebar on the sorting key of the table, Screen Reader is not providing the sorting information. Instead Screen Reader announce Space only.
 
#### Expected behavior:
 
On the Invitation page, after pressing spacebar on the sorting key of the table, Screen Reader should providing the sorting information i.e Ascending or descending. 

#### Priorities and help requested:
 
Note: 
1. This issue also exist with Chrome + JAWS.
2.  After selecting the checkbox on the table, Screen Reader is not providing the relevant information. Instead Screen Reader announce space only. (Issue convey in the attachment).

MAS Reference:

MAS 1.3.1 - Info and Relationships
 
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by dzearing** [comment]
@KevinTCoughlin @ThomasMichon would you have advice here?

**Comment by dzearing** [comment]
I am re-adding `Needs: Triage` as I am not seeing any investigation done here. This has been investigated a number of times, and I believe we're doing the right thing.

I see `aria-checked` on the highlighted element, so if the screen-reader is not reading this correctly, this might be a screenreader issue.

However the picture pasted is different than what is listed as expected. I believe if you provide aria-labels correctly on the rows, it should read changes in the sort order. I'm not sure though, and @ThomasMichon would have a better idea here since this is a common scenario in OneDrive.

**Comment by aftab-hassan** [comment]
I discussed this with @ThomasMichon offline. Similar issue has been filed a few times. This will be re-triaged as not a bug.

Adding details of our discussion below.

On hitting the `spacebar` and telling to sort by a field, we change the `state`. However, it is not possible to ask the screen reader to read a change of state (unless we put an alert at the bottom of the detailslist explaining what happened. But that can be a disruptive user experience). 

Another way to approach this might be to
The overall list could be visualized as a live region, and the entire list component could be given an `aria` label such as `List of all items in <> sorted by <>` and then wrapped around an `aria-live` region. This `aria` label of the `root` element of the `list` could then be read out to the user upon clicking the button to sort.
